school_library_decorator/student.rb

```diff
@@ -1,7 +1,9 @@
-require './person.rb'
+require './person'
 
 class Student < Person
-  def initialize(age, classroom, name = "Unknown", parent_permission = true)
+  attr_accessor :classroom
```

Comment:
- [ ] It would be better to implement the classroom method to handle the number of students in a classroom. This is where we can list all the students in a classroom
- [ ] It would be better to set the classroom for a student and also add it to the classrooms' students